lib/ranked_vote.rb

```diff
@@ -0,0 +1,8 @@
+class RankedVote
+  attr_reader :voter, :orderedCandidates
+
+  def initialize(voter, orderedCandidates)
```

Comment:
ruby の変数名はスネークケースが慣習です（他も同様）。

```suggestion
  def initialize(voter, ordered_candidates)
```

Reply:
いつもよく使う言語のそれ（`camelCase`）に完全に引っぱられてました、Ruby をほぼ書いたことがなかったので今後覚えておきます🙇‍♂️️

Reply:
ぼくも JS 書いた直後とかによくやりますw